src/base/bittorrent/customstorage.cpp

```diff
@@ -35,7 +35,7 @@
 #include "base/utils/fs.h"
 #include "common.h"
 
-#if (LIBTORRENT_VERSION_NUM >= 20000)
+#if (LIBTORRENT_VERSION_NUM >= 20004)
```

Comment:
That's not what I meant.

All conditionals like this one handle the differences between 1.2.x and 2.0.x branches in general. Why change them every time we we raise the minimum version within a branch?